zebra-chain/src/orchard/arbitrary.rs

```diff
@@ -87,6 +89,23 @@ impl Arbitrary for Signature<SpendAuth> {
     type Strategy = BoxedStrategy<Self>;
 }
 
+impl Arbitrary for VerificationKeyBytes<SpendAuth> {
+    type Parameters = ();
+
+    fn arbitrary_with(_args: Self::Parameters) -> Self::Strategy {
+        (prop::array::uniform32(any::<u8>()))
+            .prop_map(|bytes| {
+                let mut rng = ChaChaRng::from_seed(bytes);
+                let sk = SigningKey::<SpendAuth>::new(&mut rng);
+                let pk = VerificationKey::from(&sk);
+                pk.into()
+            })
+            .boxed()
```

Comment:
Minor: I'm not sure this actually possible, nor if there's a reason not to do this, but could this be generated using `halo2::pasta::pallas::Scalar::from_bytes_wide` and without having to use `rand`?

```suggestion
        any::<[u8; 64]>()
            .prop_map(|bytes| {
                let sk = pallas::Scalar::from_bytes_wide(&bytes);
                let pk = VerificationKey::from_scalar(&sk);
                pk.into()
            })
            .boxed()
```

Reply:
It is yeah, we've done it similarly for `orchard::note::Nullifier` in here 😸

Reply:
Good idea! Done in 29b13a0